config/config.go

```diff
@@ -421,8 +421,10 @@ func DefaultMempoolConfig() *MempoolConfig {
 		RecheckEmpty: true,
 		Broadcast:    true,
 		WalPath:      filepath.Join(defaultDataDir, "mempool.wal"),
-		Size:         100000,
-		CacheSize:    100000,
+		// Each signature verification takes .5ms, size reduced until we implement
+		// ABCI Recheck
+		Size:      5000,
+		CacheSize: 10000,
```

Comment:
why reduce cache size?

Reply:
Seemed kinda weird to have a 20x ratio between Size and CacheSize, though I guess there isn't a need to reduce it

Reply:
It's not about the ratio. It's about memory (total cache size) and being the first layer of protection from replay

Reply:
I think more limited defaults is fine.